modules/chapter2/pages/index.adoc

```diff
@@ -1,3 +1,3 @@
-= Chapter 2
+= Create and save a Pytorch model
 
-This is the home page of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+This chapter will cover how to create a Pytorch model with a hands-on lab session
```

Comment:
Maybe "This hands-on lab covers how to create a PyTorch model"?

Reply:
We should also mention data exploration (section 2) and that we're using RHOAI (section 1).

Reply:
Done